Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -288,6 +288,8 @@ BEGIN
     CONTROL         "Always on top",IDC_CHECK_ALWAYSONTOP,"Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,15,93,117,16
     CONTROL         "Double-click for joining channels",IDC_CHECK_DBCLICKJOIN,
                     "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,15,114,117,16
+    CONTROL         "In channels, show the name of the server in the window's title",IDC_CHECK_SERVNAMEDISP,

```

Comment:
Can we change the label to "Show only server name as window title" and default is unchecked.

Then default value for "ClientXML::GetDisplayServerName()" should be FALSE.

Otherwise the title is a big confusing